types/microsoftteams/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```

Comment:
Why is this change needed?

Reply:
Hi @ydogandjiev,

Thanks for validating this PR, this change was requested because the new types contained definitions relative to the Window object.